exporter/templates/cat1/r3/cat1.xml

```diff
@@ -22,9 +27,10 @@
   <confidentialityCode code="N" codeSystem="2.16.840.1.113883.5.25"/>
   <languageCode code="en"/>
   <!-- reported patient -->
-    {{template "_record_target.xml" .Record}}
+  {{$recordTarget := createRecordTarget .Record .CMSCompatibility}}
```

Comment:
This is where I'm saying we would be able to pass `models.Record` directly into `_record_target.xml`.